packages/@sanity/form-builder/src/inputs/arrays/ArrayOfObjectsInput/item/ArrayItem.tsx

```diff
@@ -22,6 +22,7 @@ interface ArrayInputListItemProps {
   index: number
   compareValue?: any[]
   markers: Marker[]
+  itemKey: string | undefined
```

Comment:
Nitpick: nice with `itemKey?: string` here?

Reply:
It's not equivalent. I want `itemKey` to be explicitly given, but allow its value to be `string | undefined`, but `itemKey?: string` will allow omitting the prop at the usage site.